org.tahoma2d.Tahoma2D.yaml

```diff
@@ -28,166 +24,230 @@ modules:
+      - -DWITH_OPENEXR=ON
+      - -DWITH_OPENGL=OFF
+      - -DWITH_QT=OFF
+      - -DWITH_TBB=ON
+      - -DWITH_VTK=ON
+      - -DBUILD_opencv_python2=OFF
+      - -DBUILD_opencv_python3=ON
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://github.com/opencv/opencv/archive/4.5.1.tar.gz
+        sha256: e27fe5b168918ab60d58d7ace2bd82dd14a4d0bd1d3ae182952c2113f5637513
 
-    - name: opencv
-      buildsystem: cmake-ninja
-      builddir: true
-      config-opts:
-        - -DCMAKE_BUILD_TYPE=RelWithDebInfo
-        - -DOPENCV_GENERATE_PKGCONFIG=ON
-        - -DWITH_OPENCL=ON
-        - -DWITH_OPENGL=ON
-        - -DWITH_TBB=ON
-        - -DWITH_VULKAN=ON
-        - -DBUILD_WITH_DEBUG_INFO=OFF
-        - -DBUILD_TESTS=OFF
-        - -DBUILD_PERF_TESTS=OFF
-        - -DBUILD_EXAMPLES=OFF
-        - -DINSTALL_C_EXAMPELS=OFF
-        - -DINSTALL_PYTHON_EXAMPLES=OFF
-        - -DWITH_1394=OFF
-        - -DWITH_PROTOBUF=ON
-        - -DBUILD_LIST=alib3d,core,dnn,features2d,flann,highgui,imgcodecs,imgproc,java_bindings_generator,ml,objdetect,photo,python_bindings_generator,python_tests,stitching,video,videoio
-      cleanup:
-        - /bin
-      sources:
-        - type: archive
-          url: https://github.com/opencv/opencv/archive/4.2.0.tar.gz
-          sha256: 9ccb2192d7e8c03c58fee07051364d94ed7599363f3b0dce1c5e6cc11c1bb0ec
-        - type: archive
-          url: https://github.com/opencv/opencv_contrib/archive/4.2.0.tar.gz
-          sha256: 8a6b5661611d89baa59a26eb7ccf4abb3e55d73f99bb52d8f7c32265c8a43020
-    
-    - name: tahoma2d
-      buildsystem: cmake
-      build-commands:
-        #- install -Dm 644 -t /app/share/applications com.tahoma2d.Tahoma2D.desktop
-        #- install -Dm 644 -t /app/share/icons/hicolor/256x256/apps com.tahoma2d.Tahoma2D.png
-      builddir: true
-      subdir: toonz/sources
-      config-opts:
-        - -DCMAKE_BUILD_TYPE=RelWithDebInfo
-        - -DWITH_TRANSLATION=OFF
-        - -DBOOST_ROOT=/app/include/boost
-      sources:
-        - type: archive
-          url: https://github.com/tahoma2d/tahoma2d/releases/download/v1.4/Tahoma2D-linux.tar.gz
-          sha256: 39616a1157f1fcf250914d737128c60a1e202f93fd91cbb962e4b1b9a07d7c22
-        - type: shell
-          commands:
-            - cd thirdparty/tiff-2.0 &&
-              ./configure --with-pic --disable-jbig &&
-              make -j $FLATPAK_BUILDER_N_JOBS
-        - type: file
-          path: com.tahoma2d.Tahoma2D.png
-        - type: file
-          path: com.tahoma2d.Tahoma2.metainfo.xml
-        - type: file
-          path: com.tahoma2d.Tahoma2D.desktop
+  - name: libgphoto2
+    buildsystem: simple
+    build-commands:
+      - autoreconf --install --symlink
+      - ./configure --prefix=/app
+      - make
+      - make install
+    sources:
+      # Branch: tahoma2d-version
+      - type: archive
+        url: https://github.com/tahoma2d/libgphoto2/archive/cb431893a4736853b401d19e66aa48e9f4cecd71.tar.gz
```

Comment:
what is this fork?

Reply:
Tahoma2D is my application and i need to use a customized version of libgphoto2 for my application. I maintain a fork of the original and apply my changes on top of it in a separate branch within the fork for future upgrades.

Reply:
As someone who field question for libgphoto2 support with users that use some obscure app with libgphoto2 that never get upgrade and ask why it doesn't work, I beg you to not do that. Use official releases.
It's a terribly bad practice.

Reply:
Sorry, I will not change this. 

For now, I need the current customizations and I need to control when I upgrade the libraries used by my application, not when they want to upgrade. 

I also try to avoid deviating from the official releases, but in a few rare cases I cannot help it.

Reply:
and what are these customization ? I don't remember you submitting anything.

Also at no point this automatically upgrade. the version is stuck to whatever you choose.

Reply:
I never submitted the changes to because I needed them for how I need to package things depending on the OS. I use libgphoto2 in macOS and Windows (not officially supported) builds also.

I did think about submitting issue/request to have a change made (not necessarily how I changed it) so I could do away with most of my changes.   Just never got around to it and wasn't going to hold up my development to wait for it to be fixed.

Having said that, I just reviewed my customizations to libgphoto2 and they were specifically for macOS builds and Windows compiling so I can switch to use official release version for Linux.  I prefer to build all my platform builds from the same source but I will make the exception in this case since there were no specific Linux customizations.